.env.example

```diff
@@ -32,3 +32,12 @@ HSES_DATA_PASSWORD=password
 # In production, Sequelize instance is created with a postgres URI.
 # This URI is automatically dropped into the cloud.gov environment as the env variable DATABASE_URL
 DATABASE_URL=secret
+# Local development variables to support s3 endpoint development and testing
+LOCAL_DEV=true
+# Uncomment the following if you aren't using docker for development. Note: This requires minio to be running locally on port 9000
+# S3_ENDPOINT=http://localhost:9000
+S3_BUCKET=ttadp-test
+AWS_SECRET_ACCESS_KEY=wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
```

Comment:
Can we make these more obviously fake? Maybe just the `EXAMPLEKEY` without the characters in front?

Reply:
Yes, I have updated the .env.example file.